VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitVideoFragment.java

```diff
@@ -216,6 +218,7 @@ public void setUserVisibleHint(boolean isVisibleToUser) {
         if ( ViewPagerDownloadManager.instance.inInitialPhase(unit) )
             return;
         if (isVisibleToUser) {
+            checkVideoStatus(unit);
```

Comment:
OK, since auto-play is now being handled by `PlayerFragment`, I guess this is fine as it is.